internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -478,6 +478,7 @@ type DeploymentWithStats struct {
 	Running   uint32
 	Error     uint32
 	Completed uint32
+	Stopped   uint32
```

Comment:
Where is this value changed?

Reply:
it's written to into in  `listDeploymentsWithStats`. When i did it there i forgot to do it here, this fixes that.